server/routes/immediate-needs-report/immediateNeedsReportController.ts

```diff
@@ -39,39 +33,60 @@ export default class ImmediateNeedsReportController {
       // from the API, or if this is a new assessment from the config
       const defaultVersion = existingAssessmentVersion || configVersion
 
+      // Get the CHECK_ANSWERS page to check if there's anything to pre-populate the cache with
+      const apiAssessmentPage = await this.rpService.getAssessmentPage(
+        prisonerData.personalDetails.prisonerNumber as string,
+        pathway as string,
+        'CHECK_ANSWERS',
```

Comment:
small one, but we have the string `'CHECK_ANSWERS'` it quite a lot of places, might be worth making a constant file and sticking it in there